### DISCUSSION TITLE
build from source with Conda, ctest failed

### DISCUSSION THREAD

**Discussion by blazerye** [rdkit/rdkit#4265] [Q&A]
I want to build from source with conda, 'make' and 'make install' success but ctest failed, how to fix it? and how to use rdkit in python?

https://github.com/rdkit/rdkit/blob/master/Docs/Book/Install.md#linux-x86_64-python-3-environment

cmake success:
![image](https://user-images.githubusercontent.com/16797849/122662939-71a1e880-d1c9-11eb-973a-c988019149e4.png)

make success:
![image](https://user-images.githubusercontent.com/16797849/122663174-18d34f80-d1cb-11eb-9859-01dc9bd11f6e.png)

make install success:
![image](https://user-images.githubusercontent.com/16797849/122663191-37394b00-d1cb-11eb-9830-e1e33a48b65a.png)
![image](https://user-images.githubusercontent.com/16797849/122663198-3f918600-d1cb-11eb-9dac-b42d4e1e24d0.png)

ctest failed:
![image](https://user-images.githubusercontent.com/16797849/122663139-e6c1ed80-d1ca-11eb-8e2f-64cb9a22d954.png)

use rdkit in python failed:
![image](https://user-images.githubusercontent.com/16797849/122663215-6354cc00-d1cb-11eb-861c-0a6c1abfddba.png)

**Comment by ptosco** [comment]
1. Check that the Python version picked up by `cmake` is the same as the one you are using in the `rdkit_source` conda environment (you can check this with `which python`)
2. Run one of the failing Python tests in verbose mode to see what the problem is:
```
ctest -V -I 9,9
```
3. You also seem to have some C++ tests failing; run one of those in verbose mode as well:
```
ctest -V -I 8,8
```
